.github/workflows/BundleStaticLibs.yml

```diff
@@ -106,6 +106,9 @@ jobs:
   bundle-mingw-static-lib:
     name: Windows MingW static libs
     runs-on: windows-2019
+    env:
+      ENABLE_EXTENSION_AUTOLOADING: ${{ inputs.build_static_lib_mode }}
+      ENABLE_EXTENSION_AUTOINSTALL: ${{ inputs.build_static_lib_mode }}
```

Comment:
I think those do not achieve the goal here or at very least they are confusing, given build_static_lib_mode is not yet defined.

I would put those into actions/build_extensions, and there they can be dependent on `inputs.build_static_lib_mode` OR if you prefer to have them here just define them to be 1.